src/ucp/core/ucp_worker.inl

```diff
@@ -45,11 +45,12 @@ ucp_worker_get_name(ucp_worker_h worker)
  * @return endpoint by a pointer received from remote side
  */
 static UCS_F_ALWAYS_INLINE ucp_ep_h
-ucp_worker_get_ep_by_ptr(ucp_worker_h worker, uintptr_t ep_ptr)
+ucp_worker_get_ep_by_lkey(ucp_worker_h worker, ucs_ptr_map_key_t ep_key)
```

Comment:
ep_key->ep_lkey in the args.

Reply:
IMO the func name is clean enough to specify that its arg key is local

Reply:
naming in the args makes the code easier to read and understand.